Makefile

```diff
@@ -4,7 +4,13 @@ APPS = asset-manager content-store govuk-content-schemas government-frontend \
 	collections frontend publisher calendars \
 	manuals-publisher manuals-frontend whitehall
 
-TEST_CMD = docker-compose run publishing-e2e-tests bundle exec rspec
+DOCKER_COMPOSE_CMD = docker-compose -f docker-compose.yml
+
+# Currently running in CI?
+ifndef ORIGIN_COMMIT
```

Comment:
I think you want to go for a different env var here which is always set by jenkins. This one is only set when the tests are triggered by a different build (you'll see it wasn't used for the build for this)

Looking at the one set here on https://wiki.jenkins.io/display/JENKINS/Building+a+software+project maybe JENKINS_URL is the most explicit inidicator that you're on CI or we could set something in the Jenkinsfile.